plugins/modules/filesystem.py

```diff
@@ -200,13 +239,19 @@ def get_fs_size(self, dev):
         """
         raise NotImplementedError()
 
-    def create(self, opts, dev):
+    def create(self, opts, dev, uuid=None):
         if self.module.check_mode:
             return
 
+        if uuid and self.MKFS_SET_UUID_OPTIONS:
```

Comment:
Shouldn't this code (and the below) also check whether the filesystem actually supports the `uuid` option (and fail if not, if `uuid` is provided)?

Reply:
It's reasonable to give a feedback, not just ignore the `uuid` silently. Fail feels too harsh, but certainly it draws attention. Thanx!

Reply:
If the filesystem exists and you try to change the UUID it also fails, so I guess failing here also makes sense.